app/views/api/v2/bookmarks/update.json.rabl

```diff
@@ -1,8 +1,7 @@
 object @bookmark
-attributes :name, :controller, :query, :public, :id
+attributes :name, :controller, :query, :public, :id, :owner_id, :owner_type
```

Comment:
No, this file had an update template already therefor I didn't want to take attr off so I've just added the ones missing from the show.